test/bdd/fixtures/.env

```diff
@@ -27,7 +27,6 @@ COUCHDB_KMS_PORT=5984
 COUCHDB_SHARED_PORT=5986
 
 CAS_TYPE=ipfs
-# If using CID_VERSION=0, then in create_activity.json
-# you have to change all occurances of bafkreiduenhjrl7hjgh3lwxr6nvmfv4kzqzzizhzkbydxdabtcjptavzbm to QmbF6AKvzZ2fV7GubWq9v2P6qLMXyL6Y4ebZY8hrBng7QU
-# for the BDD tests to pass
-CID_VERSION=1
+
+CID_VERSION_DOMAIN1AND3=1
```

Comment:
I would configure this per domain (CID_VERSION_DOMAIN1=1 and CID_VERSION_DOMAIN3=1), but this may be more obvious when you look at docker-compose file.